src/components/button-new-source/button-new-source.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { localization } from '../../lib/localization';
+import './button-new-source.scss';
+import { ReactComponent as Icon } from '../../assets/img/icon-add-cicle-sm.svg';
+
+export const ButtonNewSource = (props): JSX.Element | null => (
+  <div className="row">
+    <button
+      type="button"
+      aria-haspopup="true"
+      aria-expanded="false"
+      className="fdk-btn-no-border color-blue-dark mb-2"
+      onClick={props.handleClick}
+    >
+      {/* 
+      Example of svg in tag:
+      <Icon className="fdk-icon-add" />  
+      */}
+
+      <svg className="fdk-icon-add mr-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
Bruk Icon i stedet for hele svg - den er jo allerede importert

Reply:
Outdated